frontend/tests/controllers/UserRegistrationTest.php

```diff
@@ -186,4 +186,163 @@ public function testUserParentalTokenNotGeneratedDueInvalidParameters() {
             $this->assertSame('parameterInvalid', $e->getMessage());
         }
     }
+
+    /**
+     * User13 can't perform the restricted actions
+     * An exception throws
+     */
+    public function test1User13NotAbleToPerforSomeActions() {
+        $under13BirthDateTimestamp = strtotime('-10 years');
+        $randomString = \OmegaUp\Test\Utils::createRandomString();
+      //Created User13
+        \OmegaUp\Controllers\User::apiCreate(
+            new \OmegaUp\Request([
+              'username' => $randomString,
+              'password' => $randomString,
+              'parent_email' => $randomString . '@' . $randomString . '.com',
+              'birth_date' => $under13BirthDateTimestamp,
+            ])
+        );
+
+        $contestData = \OmegaUp\Test\Factories\Contest::getRequest(
+            new \OmegaUp\Test\Factories\ContestParams(
+                ['admissionMode' => 'private']
+            )
+        );
+        $r = $contestData['request'];
+        $identity = \OmegaUp\DAO\Identities::findByUsername($randomString);
+        $identity->password = $randomString;
```

Comment:
the best strategy here is to use

```suggestion
        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
            new \OmegaUp\´Test\Factories\CreateUserParams([
                'birth_date' => strtotime('-10 years'),
            ]),
        );
```

Reply:
It is giving Error!!

Reply:
I changes CreateUserParams to UserParams